tests/container_disk_test.go

```diff
@@ -146,6 +147,35 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				Expect(startedVMI.Spec).To(Equal(vmi.Spec))
 			})
 		})
+		Context("[Serial]should obey the disk verification limits in the KubeVirt CR", func() {
+			It("disk verification should fail when the memory limit is too low", func() {
+				By("Reducing the diskVerificaton memory usage limit")
+				kv := util.GetCurrentKv(virtClient)
+				kv.Spec.Configuration.DeveloperConfiguration = &v1.DeveloperConfiguration{
+					DiskVerification: &v1.DiskVerification{
+						MemoryLimit: resource.NewScaledQuantity(42, resource.Kilo),
+					},
+				}
+				tests.UpdateKubeVirtConfigValueAndWait(kv.Spec.Configuration)
+				Expect(err).To(BeNil())
+
+				By("Starting the VirtualMachineInstance")
+				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
+				_, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(vmi)
+				Expect(err).To(BeNil())
+				By("Checking that the VMI failed")
+				Eventually(func() bool {
+					vmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+					Expect(err).ToNot(HaveOccurred())
+					for _, condition := range vmi.Status.Conditions {
+						if condition.Type == v1.VirtualMachineInstanceSynchronized && condition.Status == k8sv1.ConditionFalse {
```

Comment:
One small request, could you check the content of the condition? There could be all kinds of errors reported there.

Reply:
Do you mean the error message? I don't think I can, the error message may go from being unable to load a shared lib, to a segfault or to a memory allocation error so this test could easily become flaky because of false negatives.

The test itself runs serially so whatever error there is in the condition we should fail anyway I think.

Reply:
One common solution for that which we use all over the place is this: `return fmt.Error("failed to invoke virt-chroot: %v", err)` instead of a plain `return err`. This helps debugging and you can also look for it.

Reply:
In your case you can just look for `failed to invoke qemu-img`.

Reply:
You're right! Changed it, thanks for the suggestion!